liferay-echo/liferay-portal
# LPS-178946 AUI module liferay-portlet-segments-simulation into a js component

Motivation
=======
The motivation behind this ticket is to migrate from the usage of AUI scripts to use a js component.
https://issues.liferay.com/browse/LPS-178946

Proposed Solution
========
What we did to solve this problem to use a Liferay-frontend component in the view.jsp and change the structure of the js so it is a function component.

Steps to Verify
========
1. Create some segments and some experiences.
2. Open the simulation panel
3. Check that the segments are correctly displayed
